docs/provider/conjur.md

```diff
@@ -146,10 +171,10 @@ kubectl apply -n external-secrets -f conjur-external-secret.yaml
 # kubectl delete externalsecret -n external-secrets conjur
 ```
 
-#### Step 5: Get the K8s secret
+### Step 5: Get the K8s secret
 
-* Log in to your Conjur server and verify that your secret exists
-* Review the value of your Kubernetes secret to verify that it contains the same value as the Conjur server
+* Log in to your Conjur server and verify that your secret exists.
```

Comment:
when all bullets are phrases (not full sentences), no full stops (periods for you)